lib/elixir_sense/core/binding.ex

```diff
@@ -154,6 +154,34 @@ defmodule ElixirSense.Core.Binding do
     end
   end
 
+  # dependency injection
+  def do_expand(env, {:call, {:atom, Application}, fun, args}, stack)
+      when fun in ~w(get_env get_env! fetch_env fetch_env!)a do
+    try do
+      mod = apply(Application, fun, Enum.map(args, &elem(&1, 1)))
```

Comment:
Don't we need to expand args before passing it to apply?

Reply:
I don't think so. The format of args is a list of keywords, where the key is `:atom` and the value is a proper value, like this: `[{:atom, :elixir_sense}, {:atom, :some_attribute}, {:atom, MyModule.InnerModule}]`

So calling expand for each value would return the value itself. Is there anything I may be missing?

Reply:
Not necessarily. There may be more complicated expressions (attributes, calls, etc.). That's why we need to expand. I would also check if expanded values are atoms (at least for arg 0 and 1, not sure about 2)

Reply:
Makes sense, I didn't think of that. Added in https://github.com/elixir-lsp/elixir_sense/pull/133/commits/4efd7cd3c658aa1642c14c021f93c908ab19c004

Thanks!

Reply:
Just noticed that I didn't implement the check you suggested. Do you think it's necessary? As this part is wrapped in a `try-rescue` block, if anything bad happens we're safe.